docs/source/FAQ/data_manager_migration.rst

```diff
@@ -0,0 +1,145 @@
+=============================
+Migrating to Use Data Manager
+=============================
+
+Summary
+=======
+
+This guide provides users of Citrine Python background and instructions for migrating code to take
+full advantage of Data Manager features and prepare for the future removal of endpoints that will
+occur with Citrine Python v4.0.
+
+The key change will be that :py:class:`Datasets <citrine.resources.dataset.Dataset>` are now assets
+of :py:class:`Teams <citrine.resources.team.Team>`, rather than
+:py:class:`Projects <citrine.resources.project.Project>`. The bulk of code changes will be
+migrating calls that access collections of data objects and Datasets from a Project-based method to
+a Team or Dataset-based method.
```

Comment:
```suggestion
The key change will be that :py:class:`Datasets <citrine.resources.dataset.Dataset>` are now assets
of :py:class:`Teams <citrine.resources.team.Team>`,
rather than :py:class:`Projects <citrine.resources.project.Project>`.
The bulk of code changes will be migrating calls that access collections of data objects and Datasets from a Project-based method to a Team or Dataset-based method.
```